.github/workflows/blade-pr-title-check.yml

```diff
@@ -0,0 +1,40 @@
+name: Blade PR Title Check
+
```

Comment:
why not add this check inside `validate` itself? do we really need a separate workflow?

Reply:
Need a separate workflow for this because the trigger types are different. If we keep the same workflow as blade-validate, rest of the validate jobs will get triggered even if the PR title is changed

Reply:
curious - open, edited, synchronize are the types that get triggered for validate too since we don't specify the types for it. which to me seems like these types are subset of validate workflow already.

Reply:
I think it doesn't get triggered with `edited` which is what we need for checking the PR title. Initially I started with keeping it in the same workflow but had to add these types for it to detect title changes